test/utils.spec.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-env mocha */
+'use strict'
+
+const expect = require('chai').expect
+
+const utils = require('../src/utils')
+
+describe('utils', () => {
+  it('randomSeqno', () => {
+    const first = utils.randomSeqno()
+    const second = utils.randomSeqno()
+
+    expect(first).to.have.length(20)
+    expect(second).to.have.length(20)
+    expect(first).to.not.eql(second)
+  })
+
+  it('msgId', () => {
+    expect(utils.msgId('hello', Buffer.from('world'))).to.be.eql('hello776f726c64')
```

Comment:
This was copied over from `js-libp2p-floodsub`. I think this test should be moved to `js-libp2p-pubsub` since we are also moving `utils.js` over there.